processor/src/main/java/com/blarley/rxretrobusseed/annotationprocessor/processor/RxRetroBusAnnotationProcessor.java

```diff
@@ -107,7 +111,7 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
                     Publish annotation = method.getAnnotation(Publish.class); //TODO: Build a model for this and pass that into the method
 
                     builder.append("\t\tbus.addObservable(client." + methodName + "(")
-                            .append(parameters)
+                            .append(args)
```

Comment:
Parameter types should not be included in the method invocation. `parameter` includes the param types, so it has been replaced with `args`, which does not include the types.